docs/user-guides/community/prompt-security.md

```diff
@@ -0,0 +1,38 @@
+# Prompt Security Integration
+
+[Prompt Security AI](https://prompt.security/?utm_medium=github&utm_campaign=nemo-guardrails) allows you to protect LLM interaction. This integration enables NeMo Guardrails to use Prompt Security to protect input and output flows.
+
+## Setup
+
+1. Ensure that you have access to Prompt Security API server (SaaS or on-prem).
+
+2. Update your `config.yml` file to include the Private AI settings:
+
+```yaml
+rails:
+  input:
+    flows:
+      - protect prompt
+  output:
+    flows:
+      - protect response
+```
+
+Set the `PS_PROTECT_URL` and `PS_APP_ID` environment variables.
```

Comment:
We are assuming the user that is reading this doc already understand what these variables are, I'd expect the environment variables to briefly referenced in the community doc. Others are not doing this because the environment variables are mostly API_KEY and the user has no hard time understanding of where to find it.